redirects/16.0.txt

```diff
@@ -132,13 +132,17 @@ applications/inventory_and_mrp/inventory/management/planning/scheduled_dates.rst
 applications/inventory_and_mrp/inventory/shipping/operation.rst applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping.rst                                                                  # /inventory_and_mrp/inventory/shipping/operation -> /inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping
 applications/inventory_and_mrp/inventory/shipping/operation/invoicing.rst applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/invoicing.rst                                              # /inventory_and_mrp/inventory/shipping/operation/* -> /inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/*
+applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/invoicing.rst applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/invoicing.rst                          # /inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/* -> /inventory_and_mrp/inventory/shipping_receiving/setup_configuration/*
 applications/inventory_and_mrp/inventory/shipping/operation/multipack.rst applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/multipack.rst                                              # /inventory_and_mrp/inventory/shipping/operation/* -> /inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/*
+applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/multipack.rst applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/multipack.rst                          # /inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/* -> /inventory_and_mrp/inventory/shipping_receiving/setup_configuration/*
 applications/inventory_and_mrp/inventory/shipping/operation/labels.rst applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/labels.rst                                                    # /inventory_and_mrp/inventory/shipping/operation/* -> /inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/*
-applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/labels.rst applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/labels.rst                                # /inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/labels -> /inventory_and_mrp/inventory/shipping_receiving/setup_configuration/labels
+applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/labels.rst applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/labels.rst                                # /inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/* -> /inventory_and_mrp/inventory/shipping_receiving/setup_configuration/*
```

Comment:
Nitpick– no need to change this prior redirect's comment, as it was part of the https://github.com/odoo/documentation/pull/9534 PR.
```suggestion
applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/labels.rst applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/labels.rst                                # /inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/labels -> /inventory_and_mrp/inventory/shipping_receiving/setup_configuration/labels
```